components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -236,28 +239,52 @@ class FennecMigrator private constructor(
      * @return A deferred [MigrationResults], describing which migrations were performed and if they succeeded.
      */
     fun migrateAsync(): Deferred<MigrationResults> = synchronized(migrationLock) {
```

Comment:
this could be `internal` now, right?

Reply:
The Gecko migration piece is still called from the app code directly.